src/helm/benchmark/run_specs/melt_run_specs.py

```diff
@@ -291,3 +293,117 @@ def get_math_spec(
         + get_generative_harms_metric_specs(),
         groups=["melt"] + groups,
     )
+
+
+@run_spec_function("melt_text_classification_vsmec")
+def get_melt_text_classification_vsmec_spec(fewshot: bool = False) -> RunSpec:
+    scenario_spec = ScenarioSpec(
+        class_name="helm.benchmark.scenarios.melt_scenarios.MELTTextClassificationVSMECScenario"
+    )
+
+    instruction = (
+        "Hãy phân loại cảm xúc của bình luận sau vào một trong các nhóm: "
+        "sadness, surprise, disgust, fear, anger, enjoyment, other."
+    )
+
+    adapter_spec = get_generation_adapter_spec(
+        instructions=instruction,
+        input_noun="Bình luận",
+        output_noun="Loại cảm xúc",
+        max_tokens=50,
+        max_train_instances=5 if fewshot else 0,
+        multi_label=False,
+    )
+
+    return RunSpec(
+        name=f"melt_text_classification_vsmec:fewshot={fewshot}",
+        scenario_spec=scenario_spec,
+        adapter_spec=adapter_spec,
+        metric_specs=get_exact_match_metric_specs() + get_bias_metric_specs() + get_classification_metric_specs(),
```

Comment:
I don't think `get_bias_metric_specs()` does anything in this situation; you should probably remove it.